python/lsst/ts/hvac/hvac_csc.py

```diff
@@ -31,14 +31,22 @@
 from .config_schema import CONFIG_SCHEMA
 from . import __version__
 from lsst.ts import salobj
-from lsst.ts.hvac.hvac_enums import CommandItem, HvacTopic, TelemetryItem
+from lsst.ts.hvac.hvac_enums import (
+    CommandItem,
+    HvacTopic,
+    TelemetryItem,
+    TOPICS_ALWAYS_ENABLED,
+    TOPICS_WITHOUT_CONFIGURATION,
+)
```

Comment:
I think relative imports (e.g. `from .hvac_enums import...`) are a bit safer, because you use `__init__` to hoist names up a level, which coupled with absolute imports in library code can lead to circular import issues. But if you haven't run into those then it's probably OK.

Reply:
I made all imports relative where I think they can be.